libs/model-assessment/src/lib/ModelAssessmentDashboard/ModelAssessmentDashboard.tsx

```diff
@@ -384,8 +387,8 @@ export class ModelAssessmentDashboard extends CohortBasedComponent<
                         theme={this.props.theme}
                       />
                     )}
-                    <CausalInsightsTab />
-                    <CounterfactualsTab />
+                    {/* <CausalInsightsTab />
```

Comment:
remove?

Reply:
We do want this there, but it's currently crashing the app because it's not return anything. I'd like to leave it here as a placeholder for @zhb000 / @zhb789 if possible

Reply:
I added a comment so that it doesn't just look like we forgot about it.